_history

```diff
@@ -0,0 +1,3 @@
+simulate_queue_worker
+simulate_queue_worker
+exit
```

Comment:
Hey @GilbertCherrie , should we remove these?

Reply:
Should be fixed now if you can review again when you have time